pkg/handlers/generic/mutation/handlers.go

```diff
@@ -41,6 +41,6 @@ func MetaMutators(mgr manager.Manager) []mutation.MetaMutator {
 		// Containerd restart and readiness altogether could take ~5s.
 		// We want to keep patch independent of each other and not share any state.
 		// Therefore, We must always apply this patch regardless any other patch modified containerd configuration.
-		containerdrestart.NewPatch(),
+		containerdapplypatchesandrestart.NewPatch(),
```

Comment:
Could you add a note about applying patches to the comment above this? Because the comment hasn't changed, I can't make a suggestion to it in GitHub.

Instead of 

```
		// Some patches may have changed containerd configuration.
		// We must restart containerd for the configuration to take effect.
		// Therefore, we must apply this patch last.
```

I suggest this:

```
		// Some patches may have changed containerd configuration.
		// We write the configuration changes to disk, and must run a command
		// to apply the changes to the actual containerd configuration.
		// We then must restart containerd for the configuration to take effect.
		// Therefore, we must apply this patch last.
```

Reply:
Thanks! Updated.